features/query/successful_query/feature.go

```diff
@@ -0,0 +1,54 @@
+package successful_query
+
+import (
+	"context"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows: Workflow,
+	CheckResult: func(ctx context.Context, r *harness.Runner, run client.WorkflowRun) error {
+		var qRes int
+		q1, err := r.Client.QueryWorkflow(ctx, run.GetID(), run.GetRunID(), "counterQ")
+		r.Assert.NoError(err)
```

Comment:
Change these `r.Assert` to `r.Require` or do the `if !r.Assert.NoError(err) { return r.LastAssertErr }` (sorry, this is just due to the ugliness of https://pkg.go.dev/github.com/stretchr/testify/assert and https://pkg.go.dev/github.com/stretchr/testify/require)

Reply:
Thanks. For some reason it didn't seem like NoError existed in that package.. weird. Fixed.